BlueDB/src/main/java/org/bluedb/api/BlueCollectionBuilder.java

```diff
@@ -4,8 +4,10 @@
 import java.util.Collection;
 
 import org.bluedb.api.exceptions.BlueDbException;
+import org.bluedb.api.keys.BlueKey;
 
-public interface BlueCollectionBuilder<T extends Serializable> {
-	public BlueCollectionBuilder<T> withOptimizedClasses(Collection<Class<? extends Serializable>> classesToRegister);
+public interface BlueCollectionBuilder<K extends BlueKey, T extends Serializable> {

```

Comment:
Now that we have K for key I might rather see V for value instead of T

Reply:
sounds like a good idea

Reply:
given all the likely merge conflicts as well as concerns about things like documentation not matching code after merges, I'd rather wait until most of the pending pull requests are done before changing the generic parameters